{"1": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "2": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "3": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "4": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "5": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "6": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "7": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "8": {"sample_id": 0, "compiled": true, "correctness": true, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_trials": "(5 / 5)"}, "runtime": 16.2, "runtime_stats": {"mean": 16.2, "std": 1.09, "min": 15.7, "max": 20.8, "num_trials": 100, "hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0"}}, "9": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "10": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "11": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "12": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "13": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "14": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "15": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "16": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "17": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "18": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "19": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "20": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "21": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "22": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "23": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "24": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "25": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "26": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "27": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "28": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "29": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "30": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "31": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "32": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "33": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "34": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "35": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "36": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "37": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "38": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "39": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "40": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "41": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "42": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "43": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "44": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "45": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "46": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "47": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "48": {"sample_id": 0, "compiled": true, "correctness": true, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_trials": "(5 / 5)"}, "runtime": 6.27, "runtime_stats": {"mean": 6.27, "std": 0.0103, "min": 6.25, "max": 6.29, "num_trials": 100, "hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0"}}, "49": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "50": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "51": {"sample_id": 0, "compiled": false, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "compilation_error": "Error building extension 'argmax_cuda': [1/3] /usr/bin/nvcc --generate-dependencies-with-compile --dependency-output cuda.cuda.o.d -DTORCH_EXTENSION_NAME=argmax_cuda -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\\\"_gcc\\\" -DPYBIND11_STDLIB=\\\"_libstdcpp\\\" -DPYBIND11_BUILD_ABI=\\\"_cxxabi1011\\\" -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/TH -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/THC -isystem /home/ubuntu/miniconda3/envs/kernel-bench/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_89,code=compute_89 -gencode=arch=compute_89,code=sm_89 --compiler-options '-fPIC' -std=c++17 -c /home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu -o cuda.cuda.o \nFAILED: [code=2] cuda.cuda.o \n/usr/bin/nvcc --generate-dependencies-with-compile --dependency-output cuda.cuda.o.d -DTORCH_EXTENSION_NAME=argmax_cuda -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\\\"_gcc\\\" -DPYBIND11_STDLIB=\\\"_libstdcpp\\\" -DPYBIND11_BUILD_ABI=\\\"_cxxabi1011\\\" -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/TH -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/THC -isystem /home/ubuntu/miniconda3/envs/kernel-bench/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_89,code=compute_89 -gencode=arch=compute_89,code=sm_89 --compiler-options '-fPIC' -std=c++17 -c /home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu -o cuda.cuda.o \n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(57): error: namespace \"thrust\" has no member \"device_ptr\"\n          thrust::device_ptr<float> input_ptr(input.data_ptr<float>());\n                  ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(57): error: type name is not allowed\n          thrust::device_ptr<float> input_ptr(input.data_ptr<float>());\n                             ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(57): error: identifier \"input_ptr\" is undefined\n          thrust::device_ptr<float> input_ptr(input.data_ptr<float>());\n                                    ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(58): error: namespace \"thrust\" has no member \"device_ptr\"\n          thrust::device_ptr<float> max_ptr = thrust::max_element(input_ptr, input_ptr + size);\n                  ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(58): error: type name is not allowed\n          thrust::device_ptr<float> max_ptr = thrust::max_element(input_ptr, input_ptr + size);\n                             ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(58): error: identifier \"max_ptr\" is undefined\n          thrust::device_ptr<float> max_ptr = thrust::max_element(input_ptr, input_ptr + size);\n                                    ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(58): error: namespace \"thrust\" has no member \"max_element\"\n          thrust::device_ptr<float> max_ptr = thrust::max_element(input_ptr, input_ptr + size);\n                                                      ^\n\n/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu(59): error: namespace \"thrust\" has no member \"distance\"\n          int max_index = thrust::distance(input_ptr, max_ptr);\n                                  ^\n\n8 errors detected in the compilation of \"/home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/cuda.cu\".\n[2/3] c++ -MMD -MF main.o.d -DTORCH_EXTENSION_NAME=argmax_cuda -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\\\"_gcc\\\" -DPYBIND11_STDLIB=\\\"_libstdcpp\\\" -DPYBIND11_BUILD_ABI=\\\"_cxxabi1011\\\" -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/TH -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/THC -isystem /home/ubuntu/miniconda3/envs/kernel-bench/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17  -c /home/ubuntu/KernelBench/cache/qwen3_level1_run/51/0/argmax_cuda/main.cpp -o main.o \nninja: build stopped: subcommand failed.\n"}, "runtime": -1.0, "runtime_stats": {}}, "52": {"sample_id": 0, "compiled": true, "correctness": true, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_trials": "(5 / 5)"}, "runtime": 7.93, "runtime_stats": {"mean": 7.93, "std": 2.0, "min": 7.06, "max": 14.5, "num_trials": 100, "hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0"}}, "53": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "54": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "55": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "56": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "select(): index 0 out of range for tensor of size [0] at dimension 0"}, "runtime": -1.0, "runtime_stats": {}}, "57": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "58": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "59": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "61": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "63": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "64": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "65": {"sample_id": 0, "compiled": false, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "compilation_error": "Error building extension 'conv_transpose2d': [1/3] /usr/bin/nvcc --generate-dependencies-with-compile --dependency-output cuda.cuda.o.d -DTORCH_EXTENSION_NAME=conv_transpose2d -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\\\"_gcc\\\" -DPYBIND11_STDLIB=\\\"_libstdcpp\\\" -DPYBIND11_BUILD_ABI=\\\"_cxxabi1011\\\" -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/TH -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/THC -isystem /home/ubuntu/miniconda3/envs/kernel-bench/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_89,code=compute_89 -gencode=arch=compute_89,code=sm_89 --compiler-options '-fPIC' -std=c++17 -c /home/ubuntu/KernelBench/cache/qwen3_level1_run/65/0/conv_transpose2d/cuda.cu -o cuda.cuda.o \nFAILED: [code=2] cuda.cuda.o \n/usr/bin/nvcc --generate-dependencies-with-compile --dependency-output cuda.cuda.o.d -DTORCH_EXTENSION_NAME=conv_transpose2d -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\\\"_gcc\\\" -DPYBIND11_STDLIB=\\\"_libstdcpp\\\" -DPYBIND11_BUILD_ABI=\\\"_cxxabi1011\\\" -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/TH -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/THC -isystem /home/ubuntu/miniconda3/envs/kernel-bench/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_89,code=compute_89 -gencode=arch=compute_89,code=sm_89 --compiler-options '-fPIC' -std=c++17 -c /home/ubuntu/KernelBench/cache/qwen3_level1_run/65/0/conv_transpose2d/cuda.cu -o cuda.cuda.o \n/home/ubuntu/KernelBench/cache/qwen3_level1_run/65/0/conv_transpose2d/cuda.cu(58): error: identifier \"c_in\" is undefined\n                  int input_idx = ((n * in_channels) + c_in) * (in_height * in_width) + h_in * in_width + w_in;\n                                                       ^\n\n1 error detected in the compilation of \"/home/ubuntu/KernelBench/cache/qwen3_level1_run/65/0/conv_transpose2d/cuda.cu\".\n[2/3] c++ -MMD -MF main.o.d -DTORCH_EXTENSION_NAME=conv_transpose2d -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\\\"_gcc\\\" -DPYBIND11_STDLIB=\\\"_libstdcpp\\\" -DPYBIND11_BUILD_ABI=\\\"_cxxabi1011\\\" -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/TH -isystem /home/ubuntu/miniconda3/envs/kernel-bench/lib/python3.10/site-packages/torch/include/THC -isystem /home/ubuntu/miniconda3/envs/kernel-bench/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17  -c /home/ubuntu/KernelBench/cache/qwen3_level1_run/65/0/conv_transpose2d/main.cpp -o main.o \nninja: build stopped: subcommand failed.\n"}, "runtime": -1.0, "runtime_stats": {}}, "66": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "67": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "68": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "mat1 and mat2 shapes cannot be multiplied (64x2400 and 32x262144)"}, "runtime": -1.0, "runtime_stats": {}}, "69": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_issue": "Output shape mismatch: Expected torch.Size([64, 128, 130, 260]), got torch.Size([64, 64, 130, 260])"}, "runtime": -1.0, "runtime_stats": {}}, "70": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_issue": "Output shape mismatch: Expected torch.Size([8, 24, 98, 98, 98]), got torch.Size([8, 48, 98, 98, 98])"}, "runtime": -1.0, "runtime_stats": {}}, "71": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "72": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "73": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "74": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "75": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_issue": "Output shape mismatch: Expected torch.Size([16, 64, 257, 766]), got torch.Size([16, 8, 257, 766])"}, "runtime": -1.0, "runtime_stats": {}}, "76": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "78": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "79": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "80": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "81": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "82": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "83": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "84": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "85": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "86": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "87": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "88": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "89": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "90": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "91": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "92": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "correctness_issue": "Output shape mismatch: Expected torch.Size([32767, 32769]), got torch.Size([32768, 32768])"}, "runtime": -1.0, "runtime_stats": {}}, "93": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "94": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "95": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "96": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "97": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "98": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "99": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}, "100": {"sample_id": 0, "compiled": true, "correctness": false, "metadata": {"hardware": "NVIDIA A100-SXM4-40GB", "device": "cuda:0", "runtime_error": "CUDA error: no kernel image is available for execution on the device\nCUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.\nFor debu..."}, "runtime": -1.0, "runtime_stats": {}}}